src/Array.hpp

```diff
@@ -452,38 +452,26 @@ class Array : public ArrayView< T,
    * @param newdim the new size of the default dimension.
    * @param defaultValue the value to initialize the new values with.
    * @note This preserves the values in the Array.
-   * @note The default dimension is given by m_singleParameterResizeIndex.
+   * @note The default dimension is given by singleParameterResizeIndex.
    */
   LVARRAY_HOST_DEVICE
   void resizeDefault( INDEX_TYPE const newdim, T const & defaultValue )
   { resizeDefaultDimension( newdim, defaultValue ); }
 
   /**
    * @brief Sets the size of the Array to zero and destroys all the values.
-   * @details Sets the size of m_singleParameterResizeIndex to 0 but leaves the
+   * @details Sets the size of singleParameterResizeIndex to 0 but leaves the
```

Comment:
Also change these docs, to reflect that you removed `singleParameterResizeIndex`.